nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/StandardReloadComponent.java

```diff
@@ -97,10 +100,19 @@ public void reload(final ProcessorNode existingNode, final String newType, final
         // need to refresh the properties in case we are changing from ghost component to real component
         existingNode.refreshProperties();
 
+        newNode.setProperties(getPropertiesForSetting(existingNode.getProperties()));
+
         logger.debug("Triggering async validation of {} due to processor reload", existingNode);
         flowController.getValidationTrigger().trigger(existingNode);
     }
 
+    private static Map<String, String> getPropertiesForSetting(final Map<PropertyDescriptor, PropertyConfiguration> propertyMap) {
```

Comment:
Is there a reason for making this method `static`? Just for keeping the methods organized, it would be helpful to move this method after the other public methods.

Reply:
I figured static since there is nothing about the StandardReloadComponent instance that needs to be accessed.  Do you have a thought for why an instance method would be preferable?  Either way, I'll move this to the end.

Reply:
As a private method that doesn't access or class members, it seems fine to leave it static as you described. Thanks for moving it to the end.